apps/precinct-scanner/src/utils/toSentence.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+/**
+ * Joins elements of `list` along with `comma` and `and` into parts that, when
+ * displayed together, form a sentence.
+ */
+export function toSentence(
+  list: Iterable<React.ReactChild>,
+  comma = ', ',
+  and = ' and '
+): React.ReactChild[] {
+  const elements = [...list]
+
+  if (elements.length < 2) {
+    return elements
+  }
+
+  return [
+    ...elements.slice(0, -1).flatMap((element) => [element, comma]),
```

Comment:
This happens to introduce an Oxford comma where previously it wasn't there. I can remove if desired.